content/en/docs/architecture/ci-operator.md

```diff
@@ -864,3 +864,34 @@ tests:
       - name: "pipeline:bin" # the original definition of ${DEP}
         env: "DEP"
 {{< / highlight >}}
+
+### Using secrets to retrieve credentials
+
+If your job requires to authenticate to a service,
+for example to pull an image from a registry, 
+you might have the configuration available in the build cluster already.
+
+For example, if you need to authenticate to the `Red Hat` registry (`registry.redhat.io`):
+
+{{< highlight yaml >}}
+- as: my-example-job
+  steps:
+    test:
+      - as: my-example-job
+        commands: |
+          REGISTRY_AUTH_FILE=/var/run/secrets/ci-pull-credentials/.dockerconfigjson make test
+        credentials:
+          - mount_path: /var/run/secrets/ci-pull-credentials
+            name: ci-pull-credentials
+            namespace: ci
+        from: src
+        resources:
+          requests:
+            cpu: 100m
+{{< / highlight >}}
+
```

Comment:
```suggestion
This mounts the `ci-pull-credentials` secret into the test container and sets the path to the Docker config file so that registry authentication works as expected.

```